src/store/reducers/todo.js

```diff
@@ -1,15 +1,21 @@
-import { ADD_TASK } from "../constants/actionTypes";
+import { ADD_TASK } from "store/constants/actionTypes";
+import { generateTaskObject } from "utils/helper";
 
 const initialTodoState = {
-  tasks: [],
+  tasks: [generateTaskObject("Complete Initial setup of the Todo App")],
```

Comment:
This is something new to me, we may have some one-to-one discussion about this. DM me on slack at your convenient time, will try to come to your desk.